pkgs/development/libraries/science/math/rubiks/default.nix

```diff
@@ -0,0 +1,83 @@
+{ stdenv
+, fetchurl
+, fetchpatch
+, coreutils
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rubiks";
+  version = "20070912";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "http://mirrors.mit.edu/sage/spkg/upstream/rubiks/rubiks-${version}.tar.bz2";
+    sha256 = "0zdmkb0j1kyspdpsszzb2k3279xij79jkx0dxd9f3ix1yyyg3yfq";
+  };
+
+  preConfigure = ''
+    export INSTALL="${coreutils}/bin/install"
+  '';
+
+  # everything is done in `make install`
+  buildPhase = "true";
+
+  installFlags = [
+    "DESTDIR=${placeholder "out"}"
+    "PREFIX="
```

Comment:
Normally, setting the prefix is preferred.

Reply:
Okay it works if I just set

```
    "PREFIX=${placeholder "out"}"
```

But shouldn't that be done automatically anyways?

Reply:
No, it's `--prefix` that is set by default: https://nixos.org/nixpkgs/manual/#ssec-configure-phase

Does it work if you do just `"PREFIX=$out"`?

Reply:
Yes